### DISCUSSION TITLE
Use Creevey for visual tests

### DISCUSSION THREAD

**Discussion by Fiszcz** [wKich/creevey#82] [Q&A] [answered]
For example I want use your library for visual tests but on CI pipeline and I wonder if you have some way to run **creevey** for this purpose. 

Libraries from this list have explicitly support for testing pipelines (for example in Jest): https://github.com/wKich/creevey#comparison-with-other-tools

I think that **creevey** CLI should have command/option to run only tests, return only some details about tests and exit.

**Comment by wKich** [comment, marked as answer]
Hello. Yeah, Creevey allow you to run tests on CI. You can check one of the latest runs https://github.com/wKich/creevey/runs/1378692701#step:8:92 where is Creevey test it own UI.

But, Creevey don't support docker-in-docker configuration (for some CIs, you need to configure executors to use virtual machine, check circleCI config in Creevey) and don't generate JUnit report, yet.

I'll add these features a little bit later.
